packages/shared-components/LoadingIndicator/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './rotate.css';
+
+export function LoadingIndicator({ additionalStyles }) {
+  return (
+    <div style={{
+      borderTop: '3px solid gray',
+      borderRadius: '50%',
+      boxSizing: 'border-box',
+      height: '30vh',
+      width: '30vh',
+      animationName: 'rotate',
+      animationDuration: '1s',
+      animationIterationCount: 'infinite',
+      animationTimingFunction: 'linear',
+      ...additionalStyles,
+    }}
+    />
+  );
+}
+
+LoadingIndicator.propTypes = {
+  additionalStyles: PropTypes.object, // eslint-disable-line react/forbid-prop-types
```

Comment:
use `PropTypes.shape()` => It's more appropriate

Reply:
I'm not planning on specifying the type of the contents. Does it still make a difference? Can't find any discussion on this online.

Reply:
for objects, it's shape. you don't need to override eslint.
empty shape is similar to object